### PR TITLE
V1.3.0

### PR DESCRIPTION
**Pull request by jonhealy1**
Follows the release of stac-validator 3.2.0.

### CODE REVIEW

**Review by gadomski — CHANGES_REQUESTED** [review]
Can you add a links section at the bottom so these changelog headers render as links? E.g. https://github.com/olivierlacan/keep-a-changelog/blob/ee8cc0c6256716bb57af41f7e58001fb58ae193d/CHANGELOG.md?plain=1#L193-L206.

**Comment by jonhealy1** [comment]
Yes. Good idea

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is (loosely) based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
 
-## [Unreleased]
+## [v1.3.0] - 2022-09-20
```
Comment (gadomski) [inline CHANGELOG.md:8]:
```suggestion
## [1.3.0] - 2022-09-20
```

Either the `v` needs to removed from the headings, or it needs to be added to the links at the bottom.

File: CHANGELOG.md [hunk 2]
```diff
@@ -87,4 +87,18 @@ The format is (loosely) based on [Keep a Changelog](http://keepachangelog.com/)
 - Best practices check for too many links in object
 - Best practices check for summaries in collections
 - Validation from stac-validator 2.3.0  
-- Links and assets validation checks  
+- Links and assets validation checks    
+
+-------  
+### Compare Releases  
+
```
Comment (gadomski) [inline CHANGELOG.md:94]:
```suggestion
```

No header content needed, these links are not rendered.

File: CHANGELOG.md [hunk 3]
```diff
@@ -87,4 +87,18 @@ The format is (loosely) based on [Keep a Changelog](http://keepachangelog.com/)
 - Best practices check for too many links in object
 - Best practices check for summaries in collections
 - Validation from stac-validator 2.3.0  
-- Links and assets validation checks  
+- Links and assets validation checks    
+
+-------  
+### Compare Releases  
+  
+unreleased: https://github.com/stac-utils/stac-check/compare/v1.3.0...main
```
Comment (gadomski) [inline CHANGELOG.md:95]:
```suggestion
[Unreleased]: https://github.com/stac-utils/stac-check/compare/v1.3.0...main  
```

**Review by gadomski — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (jonhealy1) [reply CHANGELOG.md:94]: ah ok

**Comment by jonhealy1** [comment]
Hi @gadomski thanks for the help with this. It's much better now.

**Review by gadomski — APPROVED** [review]

**Pull request merged**